packages/react/service_worker/OidcServiceWorker.ts

```diff
@@ -239,24 +239,30 @@ const handleFetch = async (event: FetchEvent) => {
             integrity: clonedRequest.integrity,
           }).then(hideTokens(currentDatabase));
         }
-        return undefined;
+
```

Comment:
**Is there a reason why this `undefined` was here previously?**

I think this is the simplest way to fix the issue, but I don't want to break something else